ui/src/components/ErrorPanel/index.jsx

```diff
@@ -21,14 +20,6 @@ import { CONTENT_MAX_WIDTH } from '../../utils/constants';
       ...theme.mixins.link,
     },
   },
-  fixed: {
-    position: 'fixed',
-    zIndex: theme.zIndex.drawer - 1,
-    left: '50%',
-    transform: 'translateX(-50%)',
-    width: '92%',
-    maxWidth: CONTENT_MAX_WIDTH,
-  },
 }))
```

Comment:
The pinned error panel (fixed) won't work anymore. For example, if a user scrolls down the page, the error panel will scroll out of the screen instead of always staying visible on the screen.

What I am looking for is a change to this `fixed` class to make it always stay within the boundaries of the `<main/ >` tag without removing the `position: 'fixed'` property.

![Screen Shot 2019-10-28 at 8 55 50 AM](https://user-images.githubusercontent.com/3766511/67679990-c2176300-f960-11e9-972b-9ccace58a928.png)

Reply:
Okay. I've made an update. This time, the ErrorPanel is above the drawer and is always fully visible when you scroll.